tests/e2e/specs/visit_label_printing_page.js

```diff
@@ -0,0 +1,72 @@
+// https://docs.cypress.io/api/introduction/api.html
+
+describe('Label Printing page', () => {
+  beforeEach(() => {
+    cy.visit('#/label-printing')
+
+    cy.get('#barcode_input').type('aBarcode')
```

Comment:
One of the cool things I learnt recently is the use of data attributes. Data attributes are a way to separate styling from data. https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes. I have used them in newer e2e tests and I have found them useful. https://github.com/sanger/traction-ui/blob/develop/tests/e2e/specs/pacbio/pacbio_pool_create.js